### PR TITLE
chore: add web_modules and browser: true

### PR DESCRIPTION
**Pull request by thepassle**
closes https://github.com/modernweb-dev/web/issues/184
closes https://github.com/modernweb-dev/web/issues/185

### CODE REVIEW

File: packages/test-runner/src/nodeResolvePlugin.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ export function nodeResolvePlugin(
   const options: RollupNodeResolveOptions = {
     rootDir,
     extensions: ['.ts', '.tsx', '.jsx', '.mjs', '.js', '.json'],
+    customResolveOptions: {
+      moduleDirectory: ['node_modules', 'web_modules'],
```
Comment (LarsDenBakker) [inline packages/test-runner/src/nodeResolvePlugin.ts:14]: we also need to merge in `userOptionsObject. customResolveOptions` and `userOptionsObject. customResolveOptions. moduleDirectory `. Or use deepmerge :)

**Comment by LarsDenBakker** [comment]
Can you add a changeset? `yarn changeset`

**Comment by thepassle** [comment]
> Can you add a changeset? `yarn changeset`

From the root? Or from the package?

**Comment by LarsDenBakker** [comment]
From the root. We need to add a contributing.md...

**Pull request merged**